budget-app/src/main/webapp/recordSpending.html

```diff
@@ -52,6 +52,7 @@ <h1>Record your spending</h1>
           <div class="selected">Choose your budget type</div>
         </div>
 
+<<<<<<< HEAD
```

Comment:
This is how Github tells you the start of the destination code. While helpful during the merge, it should be removed before actually committing.